metadata/as.json

```diff
@@ -11,5 +12,11 @@
 		"ns99.dns.net.nz",
 		"pch.nic.as",
 		"tld.gdns.net"
+	],
+	"policies": [
+		{
+			"type": "idn-table",
+			"comment": "http://nic.as/idn/chars.cfm"
```

Comment:
@case this could be in the `value` field, because it’s the actual table.

FIWIW, several TLDs have *HTML* IDN tables.